src/omotes_simulator_core/entities/network_controller.py

```diff
@@ -105,6 +138,48 @@ def _set_producers_to_max(self) -> dict:
         producers[self.producers[0].id][PROPERTY_SET_PRESSURE] = True
         return producers
 
+    def _set_storages_discharge_to_max(self) -> dict:
+        """Method to set the storages to the max discharge power.
+
+        :return dict: Dict with key= asset-id and value=setpoints for the storages.
+        """
+        storages = {}
+        for storage in self.storages:
+            storages[storage.id] = {
+                PROPERTY_HEAT_DEMAND: storage.max_discharge_power,
+                PROPERTY_TEMPERATURE_RETURN: storage.temperature_return,
+                PROPERTY_TEMPERATURE_SUPPLY: storage.temperature_supply,
+            }
+        return storages
+
+    def _set_storages_charge_to_max(self) -> dict:
+        """Method to set the storages to the max charge power.
+
+        :return dict: Dict with key= asset-id and value=setpoints for the storages.
+        """
+        storages = {}
+        for storage in self.storages:
+            storages[storage.id] = {
+                PROPERTY_HEAT_DEMAND: storage.max_charge_power,
+                PROPERTY_TEMPERATURE_RETURN: storage.temperature_return,
+                PROPERTY_TEMPERATURE_SUPPLY: storage.temperature_supply,
+            }
+        return storages
+
+    def _set_storages_to_demand(self, power: float = 0) -> dict:
```

Comment:
This method can either set demand or supply, please rename to for example _set_storages_power